### PR TITLE
WIP: Cleanup

### PR DESCRIPTION
**Pull request by peterbogdan**
- [x]  Merge apps `app` and `authentication` together as they are similar
- [x]  rename `core` app to `asrg`
- [x]  move media folder from under `app` to under `asrg`
- [x]  split templates from `asrg` under individual apps for easier navigation
- [x] split Job Postings, Announcements and Contributors from `website` into different apps

### CODE REVIEW

File: django_app/announcements/tests.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```
Comment (alanamircruz) [inline django_app/announcements/tests.py:1]: Not sure if we need this, probably with one simple test as a starting point and not completely empty

File: django_app/asrg/settings.py [hunk 2]
```diff
@@ -34,9 +34,12 @@
...
     "django.contrib.messages",
     "django.contrib.staticfiles",
-    "app",  # Enable the inner app
     "authentication",
-    "website",
+    "announcements",
+    "job",
+    "landing_page",
+    "contributors",
+    "website",  # TODO: remove after cleanup
```
Comment (alanamircruz) [inline django_app/asrg/settings.py:42]: I think this line can be removed

File: django_app/authentication/forms.py [hunk 3]
```diff
@@ -208,3 +212,70 @@ def clean_username(self):
     #                 raise forms.ValidationError("Password is similar to username.<br>Try a different Password")
     #         else:
     #             raise forms.ValidationError("Passwords Don't match")
+
+
+class UserProfileForm(forms.ModelForm):
+    # chapter = forms.ModelChoiceField(queryset=Chapter.objects.all(),label="",empty_label="Chapter",
```
Comment (alanamircruz) [inline django_app/authentication/forms.py:218]: Remove commented lines please

**Review by alanamircruz — COMMENTED** [review]
I still need to create a backup from the current container, then test this one locally (sanity-check basically) then I will approve. Keep you posted

File: django_app/announcements/tests.py (hunk 1 above)
Comment (peterbogdan) [reply django_app/announcements/tests.py:1]: yea, these are generated automatically when you create an app I think

File: django_app/asrg/settings.py (hunk 2 above)
Comment (peterbogdan) [reply django_app/asrg/settings.py:42]: I have to run some migrations on prod and only after that I will remove the app and the folder

**Comment by peterbogdan** [comment]
@alanamircruz @brandonbarry123 @heldyboy 
If you could pull this branch locally and test and make sure that the pages work for you too that would be great before moving this to develop 🙏

**Comment by alanamircruz** [comment]
After clicking on the top-left logo, we have an old exception from JS. I fixed it in older versions, you can check that related to some imports. Please ;D

image
